crates/nu-command/src/system/benchmark.rs

```diff
@@ -16,13 +16,17 @@ impl Command for Benchmark {
     }
 
     fn usage(&self) -> &str {
-        "Time the running time of a block"
+        "Time the running time of a closure"
     }
 
     fn signature(&self) -> nu_protocol::Signature {
         Signature::build("benchmark")
-            .required("block", SyntaxShape::Block, "the block to run")
-            .input_output_types(vec![(Type::Block, Type::String)])
+            .required(
+                "closure",
+                SyntaxShape::Closure(Some(vec![SyntaxShape::Any])),
+                "the closure to run",
+            )
+            .input_output_types(vec![(Type::Closure, Type::String)])
```

Comment:
The input type in your examples is `Type::Any`/`Type::Nothing` depending on what you provide the closure in the argument block.
```suggestion
            .input_output_types(vec![(Type::Any, Type::Duration),(Type::Nothing, Type::Duration)])
```